churchslavonic.tex

```diff
@@ -7,6 +7,7 @@
 \newfontfamily\russianfontsf[Ligatures=TeX]{lmsans10-regular.otf}
 \newfontfamily\churchslavonicfont[Script=Cyrillic,Ligatures=TeX,Scale=MatchLowercase,HyphenChar=_]{PonomarUnicode.otf}
 \newfontfamily\musicFont[Scale=1.5]{MezenetsUnicode}
+\newfontfamily\demestvo[Scale=1.5,StylisticSet=2]{MezenetsUnicode}
```

Comment:
I believe this font is not in `fonts-churchslavonic`, this will not compile

Reply:
You are right, this is the font for Znamenny Notation, and it is not part of the package yet.